bin/migration-scripts/lts-to-lts/0.16-to-0.23/playbooks/tasks/rollback.yaml

```diff
@@ -239,11 +239,41 @@
 # which leads to 2 init containers wait-for-airflow-migrations(0.19.0) +
 # run-airflow-migrations when performing rollback to 0.15.7 airflow chart
 - name: Delete All Scheduler Deployemnts
-  kubernetes.core.k8s:
+  community.kubernetes.k8s:
     api_version: v1
     state: absent
-    definition: "{{ lookup('kubernetes.core.k8s', api_version='apps/v1', kind='Deployment', label_selector='component = scheduler') }}"
+    definition: "{{ lookup('community.kubernetes.k8s', api_version='apps/v1', kind='Deployment', label_selector='component = scheduler') }}"
```

Comment:
I was just running into a problem during testing another branch where this lookup was breaking when it came back empty. I was trying to come up with some logic to do the lookup, then only do this deletion when the lookup return had 0 length, but wasn't able to solve it before switching to something else. Here's the path I was taking:

```yaml
- name: "Get schedulers"
  community.kubernetes.k8s_info:
    api_version: apps/v1
    kind: Deployment
    label_selectors: 'component=scheduler'
  register: schedulers

- debug:
    var: schedulers

- debug:
    msg: "skipping schedulers because they are null"
  when: (schedulers["resources"]|length==0)
```